integration/images/volume-copy-up/Dockerfile_windows

```diff
@@ -26,8 +26,11 @@ RUN cmd.exe /c "@echo off && FOR %i in (%BUSYBOX_EXES%) do (mklink %i.exe busybo
 
 USER ContainerUser
 
-RUN cmd.exe /c mkdir C:\test_dir
-
+RUN mkdir C:\test_dir
+# Explicitly set full access rights for "CREATOR OWNER". While
+# this is set by default for Windows Server 2019, it seems that
+# on Windows Server 2022 it is not.
+RUN icacls C:\test_dir /grant "CREATOR OWNER":(OI)(CI)(IO)F /T
```

Comment:
Perhaps this should this be in the same `RUN` as the one that creates the directory? (perhaps even combine with the `RUN` below)

Reply:
Sounds good. Testing that out in my fork now.